userland/src/add_two_ints_server.cpp

```diff
@@ -0,0 +1,42 @@
+#include <algorithm>
+#include <chrono>
+#include <cmath>
+#include <iostream>
+#include <string>
+#include <sys/time.h>
+#include <thread>
+
+#include <rclcpp/rclcpp.hpp>
+
+#include <simple_msgs/AllBuiltinTypes.h>
+#include <simple_msgs/AllDynamicArrayTypes.h>
+#include <simple_msgs/AllPrimitiveTypes.h>
+#include <simple_msgs/AllStaticArrayTypes.h>
+#include <simple_msgs/Nested.h>
+#include <simple_msgs/String.h>
+#include <simple_msgs/Uint32.h>
+
+#include <userland_msgs/AddTwoInts.h>
+#include <userland_msgs/RequestId.h>
+
+void add(const std::shared_ptr<userland_msgs::AddTwoIntsRequest> req,
+         const std::shared_ptr<userland_msgs::RequestId> req_id,
```

Comment:
The `RequestId` should come from a "RPC-namespace" instead of the message package.